### PR TITLE
Tidied up some of the Elm code.

### PR DESCRIPTION
**Pull request by wolfadex**
- cleaning up `exposing (..)`
- added a couple function type definitions
- removed unused `Msg`s
- switch some `case String of ""` into `if then else`
- In `Translate.elm` line 246, use destructuring instead of `Tuple.first` and `Tuple.second`

### CODE REVIEW

File: src/elm/lib/Api.elm [hunk 1]
```diff
@@ -19,10 +19,6 @@ type alias RandomVerbData =
     }
 
 
-type alias GetResponseObjectResult =
-    Result Http.Error ResponseObject
```
Comment (3digitdev) [inline src/elm/lib/Api.elm]: Deleting this and removing it from `exposing` causes the app to not compile.  This is needed for `Translate.elm`

Comment (3digitdev) [inline src/elm/lib/Api.elm]: Deleting this and removing it from `exposing` causes the app to not compile.  This is needed for `Translate.elm`

**Review by 3digitdev — CHANGES_REQUESTED** [review]
Deleting this and removing it from `exposing` causes the app to not compile.  This is needed for `Translate.elm`

File: src/elm/lib/Api.elm (hunk 1 above)
Comment (wolfadex) [reply src/elm/lib/Api.elm]: Added that back in

**Review by 3digitdev — APPROVED** [review]

**Pull request merged**